sdk/communication/communication-chat/src/generated/src/models/index.ts

```diff
@@ -307,249 +307,172 @@ export type ChatMessageType =
-    parsedBody: ChatThreadProperties;
-  };
-};
+/** Contains response data for the getChatThreadProperties operation. */
+export type ChatThreadGetChatThreadPropertiesResponse = ChatThreadProperties;
 
 /** Optional parameters. */
 export interface ChatThreadSendTypingNotificationOptionalParams
-  extends coreHttp.OperationOptions {
+  extends coreClient.OperationOptions {
   /** Details of the typing notification request. */
   sendTypingNotificationRequest?: SendTypingNotificationRequest;
 }
 
 /** Optional parameters. */
 export interface ChatThreadListChatReadReceiptsNextOptionalParams
-  extends coreHttp.OperationOptions {
+  extends coreClient.OperationOptions {
   /** The maximum number of chat message read receipts to be returned per page. */
   maxPageSize?: number;
   /** Skips chat message read receipts up to a specified position in response. */
   skip?: number;
 }
 
 /** Contains response data for the listChatReadReceiptsNext operation. */
-export type ChatThreadListChatReadReceiptsNextResponse = ChatMessageReadReceiptsCollection & {
-  /** The underlying HTTP response. */
-  _response: coreHttp.HttpResponse & {
-    /** The response body as text (string format) */
-    bodyAsText: string;
-
-    /** The response body as parsed JSON or XML */
-    parsedBody: ChatMessageReadReceiptsCollection;
-  };
-};
+export type ChatThreadListChatReadReceiptsNextResponse = ChatMessageReadReceiptsCollection;
 
 /** Optional parameters. */
 export interface ChatThreadListChatMessagesNextOptionalParams
-  extends coreHttp.OperationOptions {
+  extends coreClient.OperationOptions {
   /** The maximum number of messages to be returned per page. */
   maxPageSize?: number;
   /** The earliest point in time to get messages up to. The timestamp should be in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`. */
   startTime?: Date;
 }
 
 /** Contains response data for the listChatMessagesNext operation. */
-export type ChatThreadListChatMessagesNextResponse = ChatMessagesCollection & {
-  /** The underlying HTTP response. */
-  _response: coreHttp.HttpResponse & {
-    /** The response body as text (string format) */
-    bodyAsText: string;
-
-    /** The response body as parsed JSON or XML */
-    parsedBody: ChatMessagesCollection;
-  };
-};
+export type ChatThreadListChatMessagesNextResponse = ChatMessagesCollection;
 
 /** Optional parameters. */
 export interface ChatThreadListChatParticipantsNextOptionalParams
-  extends coreHttp.OperationOptions {
+  extends coreClient.OperationOptions {
   /** The maximum number of participants to be returned per page. */
   maxPageSize?: number;
   /** Skips participants up to a specified position in response. */
   skip?: number;
 }
 
 /** Contains response data for the listChatParticipantsNext operation. */
-export type ChatThreadListChatParticipantsNextResponse = ChatParticipantsCollection & {
-  /** The underlying HTTP response. */
-  _response: coreHttp.HttpResponse & {
```

Comment:
@deyaaeldeen The `_response` in the client was used to get the http response here.